docs/maintenance/database.md

```diff
@@ -57,7 +57,7 @@ With the custom format from `pg_dump` (by using `-Fc`) we can now invoke `pg_res
 !!! note
     If you run into any problems while restoring, make sure that you have the database created in that Postgres instance (e.g. before restoring the jobs database the Postgres instance should have an empty database called `jobs` created there).
 
-For more information and additional options, please check the [official documentation](https://www.postgresql.org/docs/9.6/app-pgrestore.html).
+For more information and additional options, please check the [official documentation](https://www.postgresql.org/docs/11/app-pgrestore.html).<!-- TODO DOCS-612 Confirm link should be updated to the recommended version -->
```

Comment:
Same as above, there seems to be a version mismatch in the URL.

Reply:
I updated to version 12 documentation, as it's the recommended version.